core/src/main/java/com/googlecode/psiprobe/beans/ContainerWrapperBean.java

```diff
@@ -99,9 +99,8 @@ public void setWrapper(Wrapper wrapper) {
                   tomcatContainer = (TomcatContainer) obj;
                   tomcatContainer.setWrapper(wrapper);
                   break;
-                } else {
-                  logger.debug("Cannot bind " + className + " to " + serverInfo);
                 }
+                logger.debug("Cannot bind " + className + " to " + serverInfo);
```

Comment:
This is more of a style thing. I wouldn't call this necessary, but I don't oppose it.